### ISSUE TITLE
lib/private/tempmanager.php blocked by provider because of "exploding" logfile

### ISSUE THREAD

**Issue by tflidd** [owncloud/core#14153]
A user reported on the forum that his provider blocks (all-inkl.com) the file lib/private/tempmanager.php because it blows up the logfile. They provide a workaround: https://forum.owncloud.org/viewtopic.php?f=21&t=26396#p79370
I don't know if it is fixed in this commit:
https://github.com/owncloud/core/commit/45e3cbefc9e069d5c904ed8961504551a51ce735

**Comment by karlitschek** [comment]
@DeepDiver1975 Do you know where we are with that?

**Comment by DeepDiver1975** [comment]
duplicate https://github.com/owncloud/core/issues/14087

**Comment by vanMacG** [comment]
The bug is indeed a duplicate of #14087 but the solution is a different. The commit in #14087 inserts a *if ($dh) {* while in the forumlink in this issue, the provider suggests inserting a *return;*. Don't know which one is a better way to solve the bug...

**Comment by mengine23** [comment]
@DeepDiver1975 This #14087 does not fix the problem. The provider (all-inkl.com) keeps blocking the upload of the file. Just tried it a few minutes ago.

**Comment by DeepDiver1975** [comment]
@mengine23 so all-inkl.com is blocking the upload of tempmanager.php?

Better get in touch with them on what to do.

**Comment by mengine23** [comment]
@DeepDiver1975 i believe it is more a clamav issue: mod_clamav/0.11: Virus 'owncloud_overload.UNOFFICIAL'

**Comment by LukasReschke** [comment]
That's a custom signature.

**Comment by vanMacG** [comment]
As i wrote above: all-inkl recommends inserting a *return;* in the code. Don't know if this is a better solution than the commit in #14087 ...

**Comment by mengine23** [comment]
Actually it's the only working solution for oc 8 on the host.

**Comment by tchara** [comment]
Yah, their customer support included the return in line 134. Only after that I was allowed to continue using OC8 with all-inkl.com. However, I did not have the upload issues reported by mengine23 (Did you try resetting the file permissions in KAS? When they edit the file, it does not belong to your FTP user afterwards, so you need to reset the permissions in KAS!). The other solution they suggested was downgrading to OC7... O_o
